### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by JR2A-Aurora** [UndernetIRC/gnuworld#41]
*** Connected!
Program received signal SIGSEGV, Segmentation fault.
0x00007ffff77d800e in gnuworld::msg_N::Execute (this=0x5555555bce00, params=...) at libircu/msg_N.cc:110
110     if (!nickUplink->isBursting())
(gdb)

When I launch Gnuworld, and there is a P10 server connected (via Eggdrop) to my HUB server, this message appears and GNUWorld does not launch.

**Comment by MrIron-no** [comment]
What kind of server is connected to the network? I didn’t quite understand the reference to eggdrop.

Could you run gnuworld in debug mode (-c) and paste the last N message received by gnuworld before it crashed?

**Comment by JR2A-Aurora** [comment]
Yes i explained badly sorry. To give an example, it s a TCL script launched via eggdrop which creates a server, of this type: 
https://github.com/ZarTek-Creole/TCL-EvaServ/blob/main/EvaServ.tcl

[IN ]: AB S Eva.Aurora.Corsica 2 0 1735382178 J10 Y]] + :Services
[OUT]: A7AAA O #robot :Unknown server just connected: Eva.Aurora.Corsica via hub.aurora.corsica
[IN ]: Y N Eva 2 1 Eva eva.aurora.corsica +oidk C]AAAC Yuu :Eva
Segmentation fault (core dumped)

**Comment by MrIron-no** [comment]
From what I can see that script uses YXX numeric whereas the P10 standard is YYXXX. That is likely causing the issue.

Y being the server numeric and X being the client numeric.

**Comment by JR2A-Aurora** [comment]
Correct, thhks, I will modify the tcl script it will be simpler. Strange that it didn't cause a bug with the ircu

**Comment by empus** [comment]
Issue caused by P10 protocol violation from 3rd party software
